ci-operator/config/openshift/contour-operator/OWNERS

```diff
@@ -0,0 +1,23 @@
+approvers:
+- alebedev87
+- candita
+- danehans
```

Comment:
I don't know whether we still need to add Dane :(

Reply:
Yes good point